pkg/autodiscovery/common/utils/annotations.go

```diff
@@ -154,7 +156,7 @@ func ParseJSONValue(value string) ([][]integration.Data, error) {
 
 // BuildTemplates returns check configurations configured according to the
 // passed in AD identifier, check names, init and instance configs
-func BuildTemplates(adID string, checkNames []string, initConfigs, instances [][]integration.Data) []integration.Config {
+func BuildTemplates(adID string, checkNames []string, initConfigs, instances [][]integration.Data, ignoreAutodiscoveryTags bool) []integration.Config {
```

Comment:
could you update the function description.
What happens it the `intances` container the `ignoreAutodiscoveryTags` information too? 
which settings will get the precedence?

Reply:
`integration.Data` does not hold a field `ignoreAutodiscoveryTags`, only `integration.Config` does. I updated the function description though.